open_spiel/games/german_whist_foregame/german_whist_endgame.cc

```diff
@@ -0,0 +1,729 @@
+// Source Code for an Executable Generating an Endgame Tablebase for German
+// Whist
+
+#include <cassert>
+#include <thread>
```

Comment:
I did not realize you were using threads. This is not an approved header. Can you remove the dependence on threading in this implementation? From what I can tell this is only used for the end game solver? We have something similar in Bridge but it's not threaded.